### PR TITLE
Updated demos, archived old demo info

### LINKED ISSUE #20: Update "Demos" section with correct dates and info
Needs to show the updated timeline + requirements for demo/pitch nights and final showcase this year

### PR DESCRIPTION
**Pull request by Anidion**
Closes #20 
Needs confirmation of dates before merging.

### CODE REVIEW

File: src/archive/2022-2023/demos/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+At Tech Start UCalgary
+all projects are showcased to the world
+at the end of each academic year.
+In this event, people from different industries and backgrounds
+come and give us feedback on the projects,
+while also allowing you as a professional and us as a club
+to connect and network with different companies.
+
+This year's showcase is scheduled for **April 29th, 2023**.
```
Comment (Niyousha99) [inline src/archive/2022-2023/demos/README.md:11]: April 27 this year!

File: src/archive/2022-2023/demos/3/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Just remember that in the final showcase
+(and therefore in this demo),
+you'll have:
+
+- Maximum 15 minutes for your presentation.
+- Maximum 3 minutes for questions and answers.
+- Most of the audience will be completely new to your project.
+- Teams will present in the following order:
+
+  1. Bandist.
```
Comment (Niyousha99) [inline src/archive/2022-2023/demos/3/README.md:39]: Update list of teams

**Review by Niyousha99 — CHANGES_REQUESTED** [review]

File: src/archive/2022-2023/demos/3/README.md (hunk 2 above)
Comment (Anidion) [reply src/archive/2022-2023/demos/3/README.md:39]: These are in the archive, so I kept them the same as last year 👁️

File: src/archive/2022-2023/demos/README.md (hunk 1 above)
Comment (Anidion) [reply src/archive/2022-2023/demos/README.md:11]: These are in the archive, so I kept them the same as last year 👁️

**Comment by Niyousha99** [comment]
Ok just so I understand clearly, the plan is to keep last year's demo info as archive and update the new sections for each demo night this year once we get closer to the dates and eventually remove the archive?

**Review by Niyousha99 — APPROVED** [review]

**Comment by Anidion** [comment]
> Ok just so I understand clearly, the plan is to keep last year's demo info as archive and update the new sections for each demo night this year once we get closer to the dates and eventually remove the archive?

Yeah, we don't necessarily have to remove it, but I just didn't want to delete them outright just yet

**Pull request merged**